cmd/kpod/spec_test.go

```diff
@@ -18,7 +18,7 @@ func TestCreateConfig_GetVolumeMounts(t *testing.T) {
 	config := createConfig{
 		volumes: []string{"foobar:/foobar:ro"},
 	}
-	specMount := config.GetVolumeMounts()
+	specMount, _ := config.GetVolumeMounts()
```

Comment:
This is just a test, so don't discard the error - just to an assert.NoError() after it

Reply:
fixed